easybuild/easyconfigs/f/FragGeneScan/FragGeneScan-1.31-GCCcore-10.2.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'FragGeneScan'
+version = '1.31'
+
+homepage = 'https://omics.informatics.indiana.edu/FragGeneScan/'
+description = "FragGeneScan is an application for finding (fragmented) genes in short reads."
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s%(version)s.tar.gz']
+checksums = ['cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c']
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [("Perl", "5.32.0")]
+
+fix_perl_shebang_for = ['*.pl']
+
```

```suggestion

prebuildopts = "make clean && "
```

The fix in #12870 should also be applied here.